pages/common/xargs.md

```diff
@@ -10,10 +10,14 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- Delete all files that start with 'M':
+- Insert arguments at chosen position, using '%' as the placeholder marker:
 
-`find . -name 'M*' | xargs rm`
+`{{arguments}} | xargs -I '%' {{command}} % {{extra_arguments}}`
 
-- Insert arguments at chosen position, using '%' as the placeholder marker:
+- Use the output of one command as arguments to another command:
 
-`{{arguments}} | xargs -I % {{command}} % {{extra_arguments}}`
+`{{command1}} | xargs {{command2}}`
```

Comment:
I believe users unfamiliar with the tool should read the man pages. `tldr` in my opinion is for people who already know the tool, but are not using it often enough to be worth remembering it. At least that is how I use it, more like buffed cheatsheet.

Please correct me if I am wrong.

For the other point, I believe having `{{command1}}` instead of `{{arguments}}` is clearer.